NBitcoin/Crypto/Hashes.cs

```diff
@@ -12,6 +12,8 @@
 
 #if !NONATIVEHASH
 using System.Security.Cryptography;
+using NBitcoin.Crypto.digests; // use managed SHA implementations
+using NBitcoin.BouncyCastle.Crypto.Parameters;  // we'll need this to create HMACs based o the managed SHAs.

```

Comment:
introduced 1 warning here: `The using directive for 'NBitcoin.BouncyCastle.Crypto.Parameters' appeared previously in this namespace`

Reply:
I'll fix it